e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -47,3 +59,110 @@ fixture.disablePageReloads`CheckBox`
     await createWidget('dxCheckBox', { value: undefined, text: 'indeterminate', rtlEnabled: true }, '#indeterminateRTL');
   });
 });
+
+test('Checkbox configuration, different Checkbox icon sizes', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'CheckBox configurations different iconSizes.png', { element: '#container', shouldTestInCompact: true });
+
+  await testScreenshot(t, takeScreenshot, 'CheckBox configurations different iconSizes.png', { element: '#container', theme: getDarkThemeName() });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  for (const iconSize of [20, undefined, 25]) {
+    for (const text of [undefined, 'label', 'one two three']) {
```

Comment:
i don't think testing different text here is necessary, you test it in the next test where you check different widths and text cropping